### PR TITLE
Krypto_gapps

### PR DESCRIPTION
**Pull request by ihsan4**
Hey nik! Thanks for all the hard work.

### CODE REVIEW

File: 13/Krypto_gapps.config [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+>>GoogleFiles=1
+>>StorageManager=1
+>>DocumentsUIGoogle=1
+
+MarkupGoogle=1
+GoogleTTS=1
+
+# Set GoogleSearch=0 if you want to skip installing all packages belonging to GoogleSearch Package
+GoogleSearch=0
+>>Velvet=1
```
Comment (ihsan4) [inline 13/Krypto_gapps.config]: velvet=0

**Comment by nikgapps** [comment]
We cannot merge the pull request due to following reasons

- Update 13/Krypto_gapps.config, You cannot enable both Pixel Setup Wizard and Setup Wizard, you can only enable one of them

Kindly make the changes and send the pull request again!

**Comment by ihsan4** [comment]
I have made the changes

**Comment by ihsan4** [comment]
Thank You!

**Pull request merged**